portfolio/src/main/webapp/script.js

```diff
@@ -52,6 +53,18 @@ function greetSelf() {
   });
 }
 
+function getComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    console.log(comments);
+    // Build the list of history entries.
```

Comment:
What is meant by "history entries" here? The comment history?

Reply:
yeah